app/src/cli.c

```diff
@@ -651,6 +651,7 @@ guess_record_format(const char *filename) {
 #define OPT_DISABLE_SCREENSAVER    1020
 #define OPT_SHORTCUT_MOD           1021
 #define OPT_NO_KEY_REPEAT          1022
+#define OPT_USE_ADB_KEYBOARD       1026
```

Comment:
Some numbers don't exist :trollface:

Reply:
> Thank you for the PR +1
> 
> Ideally, I would like the IME code to be included in scrcpy-server: [#37 (comment)](https://github.com/Genymobile/scrcpy/issues/37#issuecomment-681676085)
> 
> Moreover, this would also allow a socket connection between the server and the IME (to minimize latency).
> 
> However, until this is implemented (I have zero time to work on this currently), I think calling ADBkeyboard is an acceptable solution.
> 
> If this option is passed, I think it should also save the current IME, set the ADBKeyboard IME, and it should restore the previous one on cleanup.

All of these is right. :+1: